### ISSUE TITLE
listing a URL as an arg in launch.json immediately crashes the Electron app that I'm starting

### ISSUE THREAD

**Issue by pushkin-** [microsoft/vscode#114365]
- VSCode Version: 1.52.1
- OS Version: Windows 10 (Windows_NT x64 10.0.18363)

Steps to Reproduce:

1. add these args to your launch.json Launch configuration:
```
"args": [
	"url=https://google.com",
	"a=b",
],
```
2. Start the application
3. get error in Debug Console: Process exited with code 4294967295 (my app's main method doesn't even run)

If I delete the "https:", part the application launches fine
If I swap the order of the args ("a" first, then "url"), the application launches fine

My full configuration is:

```
{
	"type": "node",
	"request": "launch",
	"name": "Launch App",
	"program": "${workspaceFolder}/main.js",
	"sourceMaps": true,
	"cwd": "${workspaceFolder}",
	"runtimeExecutable": "${workspaceFolder}/node_modules/.bin/electron",
	"windows": {
		"runtimeExecutable": "${workspaceFolder}/node_modules/.bin/electron.cmd",
	},
	"runtimeArgs": [
		"--remote-debugging-port=9223",
		"."
	],
	"args": [
		"url=https://google.com",
		"a=b",
	],
	"protocol": "inspector",
	"outFiles": [
		"${workspaceFolder}/**/*.js"
	],
	"smartStep": true,
	"skipFiles": [
		"<node_internals>/**/*.js"
	]
},
```

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by weinand** [comment]
Since your application is crashing (and not VS Code), why do you think is this a problem with VS Code?

**Comment by pushkin-** [comment]
@weinand  Well,... good point. I guess it's strange, because I'm not passing runtime args to the Electron exe. I'm passing regular "args" which I expect VSCode will just forward to my program, but the crash happens before my main method is called, so I figured something is going on with VSCode and its forwarding.

I'm also not totally sure what process if being referred to by "process exited with code...". I assumed maybe it's the main app process, but maybe not.

And I believe I've found references in the issues list here mentioning that specific exit code.
